sql/gorm.go

```diff
@@ -3,11 +3,16 @@ package sql
 import (
 	"gorm.io/driver/postgres"
 	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
 )
 
-func GormOpen(cfg *Config) (*gorm.DB, error) {
+func GormOpen(cfg *Config, gormLoggerOn bool) (*gorm.DB, error) {
+	config := new(gorm.Config)
+	if gormLoggerOn {
```

Comment:
shouldn't it be `!gormLoggerOn`? (negation would be `if gormLoggerOff` then do the silent)

Reply:
or using a different name in the struct, like `cfg.SilentLogger`